cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,59 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('/')
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton')
+      .click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`You clicked a button`)
+    });
   });
 
   it('should have the ability to assert scheduled allert', () => {
-
+    let alertDisplayed = false;
+    cy.get('#timerAlertButton')
```

Comment:
We need to wait 5 sec to display alert. I don't see that you wait 5sec in your test